content/news/022/index.md

```diff
@@ -108,6 +107,32 @@ done by [naga], roughly 4x as fast as the C++ alternative
 [gfx-portability]: https://github.com/gfx-rs/portability
 [gfx-naga-blog]: https://gfx-rs.github.io/2021/05/09/dota2-msl-compilation.html
 
+### [Nestadia][nestadia]
+
+![Zelda running on Nestadia](img)
+
+[Nestadia] ([https://github.com/zer0x64/nestadia]) by [@zer0x64], [@junior-n30] and [@CBenoit]
+is a server-based NES emulator.  
+
+Nestadia was written as a reverse engineering and memory exploitation challenge
+for NorthSec CTF 2021, a cybersecurity competition, where the contestants were required
+to reverse-engineer the emulator and ultimately write a Tool Assisted Speedrun to run 
+arbitrary code inside a provided ROM.  
+
+After the competition, the code has been cleaned up from references to the competition 
+and open-sourced. The developers intend on fixing more bugs
+and adding more features in the near future.  
+
+Some interesting features of this emulator is its server-based nature, obviously, 
+and the no_std core which means that the emulator can be built and ran pretty much
+anywhere without much work.  
```

Comment:
Some grammar and wording changes here:

`Some interesting features of this emulator are its server-based nature, and the no_std core which means that the emulator can be built and ran pretty much anywhere without much work.`